Go/syntax_test_go.go

```diff
@@ -35,22 +34,30 @@ You may have to disable Go-specific linters when working on this file.
 //                            ^^^^^^^^^^^ comment.line.go
 
     /*
+//  ^^ comment.block.go punctuation.definition.comment.begin.go
 // ^ -comment
 //  ^^^^ comment.block.go
     comment
 //  ^^^^^^^^ comment.block.go
     */
-//  ^^ comment.block.go
+//  ^^ comment.block.go punctuation.definition.comment.end.go
 //    ^ -comment
 
     /* * */
 // ^ -comment
+//     ^ -punctuation.definition.comment.go
 //  ^^^^^^^ comment.block.go
```

Comment:
Just a tiny nit pick.

```suggestion
//  ^^^^^^^ comment.block.go
//     ^ -punctuation.definition.comment.go
```